auth/jwt/transport.go

```diff
@@ -0,0 +1,89 @@
+package jwt
+
+import (
+	"fmt"
+	stdhttp "net/http"
+	"strings"
+
+	"github.com/go-kit/kit/transport/grpc"
+	"github.com/go-kit/kit/transport/http"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	BEARER        string = "bearer"
+	BEARER_FORMAT string = "Bearer %s"
+)
```

Comment:
If these should be exported they should be called Bearer and BearerFormat and get comments. If not, they should be called bearer and bearerFormat.